scripts/im_plot.sl

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# script version: slurm
+#
+# must be run with sbatch im_plot.sl [imcalc csv path] [station file path] [output xyz dir] [srf path] [model params path] [realisation name] [management database location]
+
+#SBATCH --job-name=im_plot
+#SBATCH --account=nesi00213
+#SBATCH --partition=prepost
+#SBATCH --time=00:30:00
+#SBATCH --cpus-per-task=23
+
+if [[ ! -z ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
+    source $CUR_ENV/workflow/install_workflow/helper_functions/activate_env.sh $CUR_ENV "mahuika"
+fi
+
+CSV_PATH=$1
+STATION_FILE_PATH=$2
+OUTPUT_XYZ_DIR=$3
+
+SRF_PATH=$4
+MODEL_PARAMS=$5
+
+MGMT_DB_LOC=$6
+SRF_NAME=$7
+
+script_start=`date`
+echo "script started running at: $script_start"
+runtime_fmt="%Y-%m-%d_%H:%M:%S"
+timestamp=`date +%Y%m%d_%H%M%S`
+
+start_time=`date +${runtime_fmt}`
+echo ___im plot___
+
+python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $SRF_NAME IM_plot running
+res=`python $gmsim/visualization/im_plotting/im_plot.py $CSV_PATH $STATION_FILE_PATH --output $OUTPUT_XYZ_DIR`
+
+exit_val=$?
+
+load_python2_mahuika () {
+    # Load python2, have to do this as virtualenv points to this python
+    # verions, which is not accessible without loading
+    module load Python/2.7.14-gimkl-2017a
+
+    # Reset the PYTHONPATH
+    export PYTHONPATH=''
+
+    # PYTHONPATH (this can be removed once qcore is installed as a pip package)
+    export PYTHONPATH=/nesi/project/nesi00213/opt/mahuika/qcore:$PYTHONPATH
+
+    # PYTHONPATH for workflow
+    export PYTHONPATH=/nesi/project/nesi00213/workflow:$PYTHONPATH
+
+    # Load the virtual environment
+    source /nesi/project/nesi00213/share/virt_envs/python2_mahuika/bin/activate
+}
+
+plot_stations () {
+    if [[ -f "$f" ]]; then
+        out_dir="${f//./_}_png_stations"
+        echo "outputtig pngs to $out_dir"
+        python $gmsim/visualization/gmt/plot_stations.py $f -n 22 --srf "adfsa$SRF_PATH" --model_params $MODEL_PARAMS --out_dir $out_dir
```

Comment:
Probably can lower the number of cores requested to 4. Just because this isn't computationally demanding. This line also has a random string in the middle.